cts/cts-scheduler.in

```diff
@@ -1557,7 +1563,7 @@ class CtsScheduler(object):
             self._error("%d (of %d) tests failed (no diff results)" %
                 (self.num_failed, self.num_tests))
             if os.path.isfile(self.failed_filename):
-                os.remove(self.failed_filename)
+                shutil.rmtree(self.failed_dir)
```

Comment:
We should also remove the directory if self.num_failed == 0 above this

Reply:
That happens on line 1550. Did you mean something else?

Reply:
Nope, I missed that, thanks